roadmap.js

```diff
@@ -298,7 +298,7 @@
   function _loadTooltipModule() {
     // Be sure to change the preloads in markdown when updating url.
     // The ESM bundle of this package doesn't work with unpkg.com.
-    const module = import('https://cdn.jsdelivr.net/npm/@floating-ui/dom@1/+esm');
```

Comment:
does the "1"  here mean "1.0" (i.e. pinned to an older revision) or does it mean "get me the latest 1.x version"?

Reply:
No idea.. I just verified that the `1` didn't give me the new version that I wanted and `1.3` did.

Reply:
@RReverser is this the correctly thing to do here?

Reply:
> does the "1" here mean "1.0" (i.e. pinned to an older revision) or does it mean "get me the latest 1.x version"?

I honestly don't know; before https://github.com/WebAssembly/website/pull/283 we used unpkg.com where it wouldn't be pinned, but I'm not very familiar with what jsdelivr does.

Explicitly bumping to 1.3 sounds good.

Reply:
That said, https://cdn.jsdelivr.net/npm/@floating-ui/dom@1.3/+esm doesn't seem to work?

Reply:
Wait, I don't think this is a module you meant to bump at all. This is not wasm-feature-detect, it's just a tooltip library.

Reply:
Ooops! Wrong URL.

Actually I noticed that the existing URL seems to fetch 1.3 already: https://cdn.jsdelivr.net/npm/wasm-feature-detect@1/dist/esm/index.js

So I'm closing this..

Reply:
Just to clarify @1 will load the latest 1.x.x version https://github.com/jsdelivr/jsdelivr#npm
The HTTP headers also verify it:

```
x-jsd-version: 1.3.0
x-jsd-version-type: version
```